### DISCUSSION TITLE
Unable to join Slack workspace

### DISCUSSION THREAD

**Discussion by inactivist** [wagtail/wagtail#10096] [General]
Not sure where to mention this.  I'm unable to join the slack workspace, not sure how to proceed.

Following this link on the Support page:

![image](https://user-images.githubusercontent.com/1195074/219039223-56ddc5f1-bdea-4a0a-a187-5bbf3089222d.png)

When I try to join by clicking [Continue with Google] button, I select my google account in the Google 2FA page, then I'm redirected back to the signup page and I see the following error message -- and the list of approved domain names doesn't include gmail.com, obviously.

Is there an option join the Wagtail slack using one's existing Slack account not tied to one of the approved TLDs?  

![image](https://user-images.githubusercontent.com/1195074/219038602-2d0c0a20-69e0-42cb-9a54-d631e952ab1d.png)

**Comment by gasman** [comment]
Thanks for reporting! Visiting https://wagtail.org/slack should give you a working invitation link now.

(Annoyingly Slack has a limit on the number of people who can join under a given invitation link, so it expires every few months.)

**Comment by jsma** [comment]
I'm having a similar issue. If I recall correctly, in the past I would just follow the link from the wagtail.org site, enter my email address and name to get a code to enter. Safari usually kept me logged in though, so I haven't had to go through any authentication steps in several months. 

Now when I follow the link, choose the "continue with email" option, and enter the code I receive via email, I get this error:
<img width="600" alt="Screenshot 2023-08-12 at 11 07 45 AM" src="https://github.com/wagtail/wagtail/assets/24574/dd6d7b6a-b281-4116-a18e-2483e376a181">

I had also tried the Google option, but got the same error as the OP. I tried the Apple option, but it rejects "hide my email" addresses. I wasn't super active in Slack, but enjoyed following along the announcements etc. Thanks for any assistance!

**Reply by zerolab** [reply]
@jsma https://wagtail.org/slack is for joining. You want to use https://wagtailcms.slack.com/ for logging in
